hack/vendor.sh

```diff
@@ -55,10 +55,27 @@ clone git github.com/docker/libtrust 230dfd18c232
 
 clone git github.com/Sirupsen/logrus v0.7.2
 
-clone git github.com/go-fsnotify/fsnotify v1.2.0
+clone git github.com/go-fsnotify/fsnotify v1.0.4
 
 clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
 
+clone git github.com/philhofer/fwd 899e4efba8eaa1fea74175308f3fae18ff3319fa
+
+clone git github.com/tinylib/msgp 75ee40d2601edf122ef667e2a07d600d4c44490c
+
+clone git github.com/fluent/fluent-logger-golang v0.6.0
+
+# get Go tip's archive/tar, for xattr support and improved performance
```

Comment:
Oh, i got a mistake on rebase operation of this file. I'll rebase this again and push it.